accessing
exceptionHandler: aHandler
	
	exceptionHandler := aHandler